Update documentation (#112)
insightsengineering/r.pkg.template: README.md

> All workflows originating from this repository can be repurposed by other R package Github repositories.

Reviewer: ```suggestion
All workflows originating from this repository can be repurposed by other R package GitHub repositories.
```

> For detailed description and screenshots of workflows please refer to [this document](./workflows.md).

Reviewer: ```suggestion
For more information including detailed description and screenshots of workflows, please refer to [Workflows](./workflows.md).
```

### Pre-commit

> This repository contains an example [pre-commit] configuration.

Reviewer: Comment to line 77: [pre-commit] is a tool that uses [Git hooks] to identify and resolve simple issues before submission for code review.

Author: Updated according to the suggestion.
